### DISCUSSION TITLE
We're looking for maintainers to keep the project alive

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#731] [Announcements]
As you may know, this project/fork started because [the original project](https://github.com/0x192/universal-android-debloater) relied on 1 owner/maintainer, which eventually led to the project's death. We wanted to continue the project and thus started this fork and shortly after had [our first release](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/releases/tag/0.5.2).

However, 1 year later (time flies!) the development has stagnated quite a bit: some members just disappeared for months without any Discord communication. And yeah, of course, open-source contributions are completely voluntary. It's just that I personally value some sort of communication at least. 😉 Long story short: everyone's busy, including myself. Unfortunately I am not a developer, so I won't be able to write new code, features or complex bug-fixes myself. Pull requests need to be reviewed, too. The most important thing is that issues regarding `wgpu` will get solved, as they're pretty widespread:

- https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/issues/459
- https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/issues/383
- https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/issues/645

I do not want to let this project die, as there's clearly a need for a tool like UAD-ng ([the amount of stars we gathered in such a short time proves this](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/discussions/418#discussion-6486668)).

So to conclude: **we're in need of [Rust](https://www.rust-lang.org) developers** to fix some bugs. Features are not that important right now, as long as critical bugs are fixed so that everyone can enjoy using UAD-ng. We can add you to the team as member and communicate through either GitHub only or Discord as well (whatever you prefer of course). If you're interested, let us know by replying this announcement, or [join our Discord](https://discord.gg/CzwbMCPEZa). If you prefer a different method, send an old-skool email to anonymouswp@oxygenupdater.com.

Thanks for all the stars and support, and those who contributed of course!❤️

The UAD-ng team.

**Comment by Marekk2k** [comment]
Sad that the project has been so neglected for so long. But I sincerely hope that it will find new programmers. I really do. When it comes to debloater for android, I prefer this project, not the one from the Russian programmer from Adb Controll App.

**Comment by I-I-IT** [comment]
I am sorry I don't know Rust. You probably want to pin this and also copy the message in Issues and in the README,no one reads Github Discussions anyway.

**Reply by AnonymousWP** [reply]
Good point! I'll put it in the README as well. This is already pinned by the way. 😉

**Reply by AnonymousWP** [reply]
https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/744

**Comment by Rudxain** [comment]
- [Official learning materials](https://www.rust-lang.org/learn) (They're actually good! No need for outdated YT tutorials)
- [NoBoilerplate Rust videos](https://youtube.com/playlist?list=PLZaoyhMXgBzoM9bfb5pyUOT3zjnaDdSEP), also [available in text form](https://github.com/0atman/noboilerplate/)

**Reply by Cyborgnetical** [reply]
Im going to learn rust, ill be back lol

**Comment by AnonymousWP** [comment]
Hey @Muriilow, I see you're being very active on our repository! Would you like to join our Discord-server (see README)? I'd like you to have on our team to review some pull requests regarding packages.🙂

**Reply by Muriilow** [reply]
Whats up! Im already on the dc server, Im going to send you a message :)

**Reply by Rudxain** [reply]
I agree! I think we should add @Iamanaws as well

**Reply by Muriilow** [reply]
hes doing a lot of work ngl

**Reply by AnonymousWP** [reply]
@Iamanaws Are you in our Discord? If so, please send me a DM or tag me in one of the channels. 🙂
